packages/polaris-viz-core/src/types.ts

```diff
@@ -72,12 +72,20 @@ export interface XAxisOptions {
   labelFormatter?: LabelFormatter;
   hide?: boolean;
   allowLineWrap?: boolean;
+  label?: string;
+  lowLabel?: string;
```

Comment:
i couldn't find any reference to this `lowLabel` attribute but only for the YAxisOptions on line [654](https://github.com/Shopify/polaris-viz/pull/1720/files#diff-8b708025e4f5317813d52c35079624dcbb0e2e2797a78a8290409a9098b780d3R654) of Grid component.